guides/micronaut-openapi-generator-server/micronaut-openapi-generator-server.adoc

```diff
@@ -0,0 +1,497 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+In this guide we will write an OpenAPI definition file and then use it to generate a Java Micronaut server API.
+Then we will add internal logic to the API and test our implementation.
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+== Installing OpenAPI Generator
+
+To use OpenAPI Generator we need to install OpenAPI Generator CLI.
+
+We will use the jar option. Download it from
+https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar
+and store it in the folder that you want to use for this project.
+
+Next, open terminal in the same folder. To verify that generator works correctly run the `help` command:
+[source,bash]
+----
+java -jar openapi-generator-cli-5.4.0.jar help
+----
+
+It should provide a description and a list of commands.
+
+[NOTE]
+===============================
+All the options for installation are given at the https://openapi-generator.tech/docs/installation[OpenAPI Generator website].
+
+If you installed the generator with a packet manager or bash launcher script, simply run
```

Comment:
packet -> package

Reply:
Added short description for Homebrew and bash script.
Not sure if that is needed, as the link above takes to all the options for installation (updated its name, too).